core/build.gradle

```diff
@@ -31,4 +31,3 @@ generateLexer {
 }
 
 compileKotlin.dependsOn generateLexer
-dokkaHtml.dependsOn generateLexer
```

Comment:
Don't know, but it failed so I removed this...

Reply:
whevs, its just a compatibility fix and not the main branch